FswStressTests/Scenario01_BasicFileCreation.cs

```diff
@@ -0,0 +1,51 @@
+namespace FswStressTests;
+
+/// <summary>
+/// Scenario 1: Basic file creation monitoring
+/// Tests the simplest case - watching for a single file creation
+/// </summary>
+public class Scenario01_BasicFileCreation : IStressScenario
+{
+    public string Name => "Basic File Creation";
+
+    public async Task RunAsync()
+    {
+        var testDir = Path.Combine(Path.GetTempPath(), $"fsw_test_{Guid.NewGuid()}");
```

Comment:
prefix each created directory with something that includes the scenario name. So if any of them is not deleted, we can more easily narrow the faulty test:

```suggestion
        var testDir = Path.Combine(Path.GetTempPath(), $"fsw_test_basic_file_creation_{Guid.NewGuid()}");
```